js/src/forum/components/PostStream.js

```diff
@@ -121,24 +121,17 @@ export default class PostStream extends Component {
    * Start scrolling, if appropriate, to a newly-targeted post.
    */
   triggerScroll() {
-    if (!this.attrs.targetPost) return;
+    if (!this.attrs.targetPost || !this.stream.needsScroll) return;
```

Comment:
You might even be able to drop `!this.attrs.targetPost` if you know it's always set together with `needsScroll